dotCMS/src/main/webapp/html/portlet/ext/contentlet/contentlet_actions_inc.jsp

```diff
@@ -32,104 +33,111 @@ try{
 	}
 }
 catch(Exception e){
-	wfActions = new ArrayList();
+	wfActions = new ArrayList<>();
```

Comment:
if you want just to return an empty list, you can do something like Collections.emptyList();